lib/nodejs/vwf.js

```diff
@@ -139,19 +141,37 @@ function HandleFileRequest( request, response ) {
 // directory structure.
 // If all that fails, serve up a 404 response since the request was not handled.
 function Serve( request, response ) {
+    lobby.post( "/sim", function( req, res ) {
+        var simDirectory = "sim";
+        var data = req.body;
+        var filePath = simDirectory + "/" + data.filePathFromSim;
+        fs.writeFile( filePath, data.simData, function( err ) {
+            if ( err ) {
+                return console.error( err );
+            }
+            execFile( data.simExe, [
+                data.commandLineFlag,
+                data.filePathFromSim
+            ], {
+                cwd: simDirectory
+            }, function( err ) {
+                if ( err ) {
+                    return console.error( err );
+                }
+            } );
+        } );
+        res.sendStatus( 200 );
```

Comment:
Sending the result here will mark every request successful regardless of whether errors eventually occur. The request also may or may not complete before the `writeFile` and the `execFile` complete.